fastunit/__init__.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
May we pull fastunit dependency from GitHub repo, instead of copying it in init actions repo? Maybe we need to create Docker image with pre-installed fastunit?